unstructured/nlp/patterns.py

```diff
@@ -33,3 +33,14 @@
     "·",
 ]
 UNICODE_BULLETS_RE = re.compile(f"({'|'.join(UNICODE_BULLETS)})")
+
+IP_ADDRESS_PATTERN = "[0-9]{1,2}\.[0-9]{1,2}\.[0-9]{1,2}\.[0-9]{1,2}","[a-z0-9]{4}::[a-z0-9]{4}:[a-z0-9]{4}:[a-z0-9]{4}:[a-z0-9]{4}%?[0-9]*"
```

Comment:
For the IP address patterns, mapi id, and datetime, could you put a comment with an example of the type of string it would extract? Email I think is self-explanatory.